example_sign1_test.go

```diff
@@ -0,0 +1,71 @@
+package cose_test
```

Comment:
For what I've seen in the wild, people tend to put all examples in the same file, normally named `examples_test.go`, so examples don't pollute the file tree.

Reply:
It depends. One counterexample is https://github.com/golang/go/tree/master/src/sort

Reply:
merged all examples to `example_test.go`.